assets/javascripts/discourse/widgets/discourse-reactions-counter.js.es6

```diff
@@ -82,6 +82,7 @@ export default createWidget("discourse-reactions-counter", {
       } else {
         this.state.postId = null;
       }
+      _popperStatePanel && _popperStatePanel.update();
```

Comment:
I believe after a couple of false starts we now can use the safe navigation operator in plugins, right?

Reply:
Yes :) old habits :p